docs/supported_languages_and_lockfiles.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: page
+title: Supported Languages and Lockfiles
+permalink: /supported-languages-and-lockfiles/
+nav_order: 2
+---
+
+# Supported Languages and Lockfiles
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+## Supported lockfiles
+
+A wide range of lockfiles are supported by utilizing this [lockfile package](https://github.com/google/osv-scanner/tree/main/pkg/lockfile).
+
+|  Language  | Compatible Lockfile(s) |
+|:-----------|:------------------|
+| Elixir | `mix.lock` |
+| C/C++ | `conan.lock`|
+| Go | `go.mod` |
+| Java | `buildscript-gradle.lockfile`<br>`gradle.lockfile`<br>`pom.xml`[\*](https://github.com/google/osv-scanner/issues/35)<br>`yarn.lock` |
```

Comment:
yarn.lock is Java

Reply:
Right. yarn.lock is in the java row already.

Reply:
I assume Oliver meant its Java _script_ - it's very much not Java 😅 (it falls under the NPM ecosystem)

```suggestion
| Java | `buildscript-gradle.lockfile`<br>`gradle.lockfile`<br>`pom.xml`[\*](https://github.com/google/osv-scanner/issues/35) |
```

Reply:
Yep I meant JavaScript 🤦. My bad!